settings.gradle

```diff
@@ -57,6 +57,7 @@ include 'clients',
     'connect:runtime',
     'connect:test-plugins',
     'connect:transforms',
+    'coordinator-common',
```

Comment:
can you help me understand what this does? my understanding is we should only disclose group-coordinator-api to the clients module

Reply:
AFAIK, all the subprojects/modules for the kafka project must be declared in settings.gradle
Don't think this has anything to do with `clients` module.